openprescribing/pipeline/tests/test_fetch_and_import_ncso_concessions.py

```diff
@@ -46,7 +46,7 @@ def test_fetch_and_import_ncso_concessions(self):
         with open(os.path.join(base_path, "ncso-archive.html")) as f:
             archive_doc = bs4.BeautifulSoup(f.read(), "html.parser")
 
-        with open(os.path.join(base_path, "ncso-current.html")) as f:
+        with open(os.path.join(base_path, "ncso-current-new-format.html")) as f:
```

Comment:
I decided to add a new test file rather than replacing the old one, because it was useful to me to be able to see what the old page format looked like.  (The new one is literally a copy/paste of the current page source, with the table content replaced. I'm fairly sure this is how the previous one was generated too.)